src/LayoutManager/LayoutManager.ts

```diff
@@ -72,35 +93,26 @@ export class LayoutManager {
     this.unsubscribe(object, context);
     const disposers = [
       object.on('modified', (e) =>
-        this.performLayout({
+        target.layoutManager.performLayout({
           trigger: 'modified',
           e,
           type: LAYOUT_TYPE_OBJECT_MODIFIED,
           target,
         })
       ),
-      ...(
-        [
-          'moving',
-          'resizing',
-          'rotating',
-          'scaling',
-          'skewing',
-          'changed',
-          'modifyPoly',
-        ] as TModificationEvents[]
-      ).map((key) =>
-        object.on(key, (e) =>
-          this.performLayout({
+      ...layoutingEvents.map((key) =>
+        object.on(key, (e) => {
+          target.layoutManager.performLayout({
             trigger: key,
             e: { ...e, target: object },
```

Comment:
target here should be outside of e and called something like 'triggerSource' or something on that way.
Usually what we call e is the event that generated the call to the callback, and in this case target isn't part of it.
So adding it there is a bit confusing. I understand we want to comunicate to the dev which object moved and triggered a layout, but that can be done with the specific name.

out of scope for this pr

Reply:
Sounds correct